spec/requests/api/settings_spec.rb

```diff
@@ -38,5 +54,61 @@
 
       expect(response).to have_http_status(:not_found)
     end
+
+    it "supports multiple categories" do
+      stub_api_collection_config("settings", "categories", %w(product authentication server))
+      api_basic_authorize action_identifier(:settings, :read, :resource_actions, :get)
+
+      run_get settings_url
+
+      expect(response.parsed_body).to match(
+        "product"        => normalize_settings(Settings.product),
```

Comment:
correct, if we have a declared Settings hash (without symbol or nil data), the normalize_hash wouldn't be needed. It was written as to make sure the Settings returned is how the API would have rendered it.